datasets/noaa-ofs.yaml

```diff
@@ -35,6 +35,16 @@ Resources:
     ARN: arn:aws:sns:us-east-1:123901341784:NewOFSObject
     Region: us-east-1
     Type: SNS Topic
+  - Description: CO-OPS Operational OFS Data (Historical Retention)
+    ARN: arn:aws:s3:::noaa-nos-ofs-pds
+    Region: us-east-1
+    Type: S3 Bucket
+    Explore:
+    - '[Browse Bucket](https://noaa-nos-ofs-pds.s3.amazonaws.com/index.html)'
+  - Description: New data notifications for CO-OPS OFS, only Lambda and SQS protocols allowed
+    ARN: arn:aws:sns:us-east-1:

```

Comment:
Oh, this is missing the rest of the arn. Do you know what it is @Patrick-Keown ?